$OpenBSD$

Use SIGUSR1 in OpenBSD as abort signal
EXPERIMENTAL: use SIGUSR2 as suspend signal 
(there is mention somewhere that this is how it's done on Linux)

Index: mono/utils/mono-threads-posix-signals.c
--- mono/utils/mono-threads-posix-signals.c.orig
+++ mono/utils/mono-threads-posix-signals.c
@@ -68,6 +68,8 @@ abort_signal_get (void)
 {
 #if defined(HOST_ANDROID)
 	return SIGTTIN;
+#elif defined (__OpenBSD__)
+	return SIGUSR1;
 #elif defined (SIGRTMIN)
 	static int abort_signum = -1;
 	if (abort_signum == -1)
@@ -85,6 +87,8 @@ suspend_signal_get (void)
 {
 #if defined(HOST_ANDROID)
 	return SIGPWR;
+#elif defined (__OpenBSD__)
+	return SIGUSR2;
 #elif defined (SIGRTMIN)
 	static int suspend_signum = -1;
 	if (suspend_signum == -1)
